docs/content/misc/tasks.md

```diff
@@ -251,7 +251,6 @@ The indexing service can also run real-time tasks. These tasks effectively trans
       "maxRowsInMemory": 500000,
       "intermediatePersistPeriod": "PT10m",
       "windowPeriod": "PT10m",
-      "basePersistDirectory": "\/tmp\/realtime\/basePersist",
```

Comment:
@cheddar can you please add the explanation ? Is this because we want to consolidate all things related to task in the task working directory?